docs/website/layouts/partials/home/banner.html

```diff
@@ -3,6 +3,13 @@
 {{ $tagline     := site.Params.tagline }}
 {{ $socialMenu  := site.Menus.social }}
 {{ $docs        := "/docs/latest" }}
+{{ $introduction  := "/docs/latest" }}
+{{ $tutorial  := "/docs/latest/get-started" }}
+{{ $policyLanguage  := "/docs/latest/policy-language" }}
+{{ $kubernetes  := "/docs/latest/kubernetes-admission-control/" }}
+{{ $microservices  := "/docs/latest/http-api-authorization" }}
```

Comment:
Just clarifying that this will point to the `Envoy` tutorial after the release ?

Reply:
Yep, if we cut the release before merging I'll update the PR with all the new URLs